src/smart-components/access-management/UsersTable.js

```diff
@@ -0,0 +1,124 @@
+import React, { useState, useEffect } from 'react';
+import { useDataViewSelection, useDataViewPagination } from '@patternfly/react-data-view/dist/dynamic/Hooks';
+import { BulkSelect, BulkSelectValue } from '@patternfly/react-component-groups/dist/dynamic/BulkSelect';
+import { DataView } from '@patternfly/react-data-view/dist/dynamic/DataView';
+import { DataViewToolbar } from '@patternfly/react-data-view/dist/dynamic/DataViewToolbar';
+import { DataViewTable } from '@patternfly/react-data-view/dist/dynamic/DataViewTable';
+import { Pagination } from '@patternfly/react-core';
+import { ActionsColumn } from '@patternfly/react-table';
+import { fetchUsers } from '../../helpers/user/user-helper';
+
+const COLUMNS = ['Username', 'Email', 'First name', 'Last name', 'Status', 'Org admin'];
+
+const ROW_ACTIONS = [
+  { title: 'Add to user group', onClick: () => console.log('ADD TO USER GROUP') },
+  { title: 'Remove from user group', onClick: () => console.log('REMOVE FROM USER GROUP') },
+];
+
+const PER_PAGE_OPTIONS = [
+  { title: '5', value: 5 },
+  { title: '10', value: 10 },
+  { title: '20', value: 20 },
+  { title: '50', value: 50 },
+  { title: '100', value: 100 },
+];
+
+const OUIA_ID = 'iam-users-table';
+
+const UsersTable = () => {
+  const [users, setUsers] = useState([]);
+  const [totalCount, setTotalCount] = useState(0);
+  const [loading, setLoading] = useState(true);
+  const [error, setError] = useState(null);
+
+  const pagination = useDataViewPagination({ perPage: 20 });
```

Comment:
Can you please enable pagination params sync with URL to keep the functionality we have in the Users page?